docs/source-app/workflows/byoc/index.rst

```diff
@@ -0,0 +1,92 @@
+
+#################################
+Run Apps on your own cloud (BYOC)
+#################################
+
+**Audience:** Users looking to run Lightning Apps on their own private cloud.
+
+----
+
+*******************
+A bit of background
+*******************
+
+BYOC - Bring Your Own Cloud, is an alternate deployment model to Lightning Cloud (fully managed SaaS).
+BYOC separates the control and data plane. The data plane, that includes
+Lightning clusters, services and Lightning Apps, reside inside the user’s VPC.
+The control plane resides on Lightning Cloud.
+
+Setup begins with configuring a cloud provider (today AWS, but more are coming soon) with your personal credentials for
+delegated access and an identity provider for secure access to the data plane.
+
+Next, as part of the environment creation process, users can configure networking,
+security, and select among cluster configuration options based on their own use cases.
+
+After submitting a cluster creation request, the Lightning Control Plane creates the required cloud infrastructure on the user account. This
+sets up a new Lightning Cluster along with a Lightning Kubernetes Operator.
+
+
+*******************************
+Create a Lightning BYOC cluster
+*******************************
```

Comment:
I think we need a section here noting that cluster creation can take a long time (1+hrs)

Reply:
Updated.